akka-http-core/src/main/java/akka/http/javadsl/model/headers/HttpOriginRange.java

```diff
@@ -24,5 +24,5 @@ public static HttpOriginRange create(HttpOrigin... origins) {
      */
     @Deprecated
     // FIXME: Remove in Akka 3.0
-    public static final HttpOriginRange ALL = HttpOriginRanges.ALL;
+    HttpOriginRange ALL = HttpOriginRanges.ALL;
```

Comment:
I know it's the same in an interface but would prefer to keep the public static final here